chore(issue templete): update issue template (#4218)
emqx/emqx: .github/ISSUE_TEMPLATE/bug-report.md

---
name: Bug Report

> about: Report a bug encountered while operating EMQ X

Reviewer: ```suggestion
about: Report a bug of EMQ X
```

title: ''
labels: BUG
assignees: tigercl

---

<!-- Please use this template while reporting a bug and provide as much info 
as possible so that we can support you more efficiently. Thanks!-->

#### What happened:

> #### What you expected to happen:

Reviewer: ```suggestion
#### Expected behaviour
```

> #### How to reproduce it (as minimally and precisely as possible):

Reviewer: ```suggestion
#### How to reproduce it (prefer minimal and precise steps):
```

> #### Anything else we need to know:

Reviewer: ```suggestion
#### More information to help understand/troubleshoot
```
